tests/Issues/RuleWithConfigurationAfterSet/config/configured_rule.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Php55\Rector\String_\StringClassNameToClassConstantRector;
+use Rector\Set\ValueObject\LevelSetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->sets([
+        LevelSetList::UP_TO_PHP_81,
+    ]);
+
+    $rectorConfig->ruleWithConfiguration(StringClassNameToClassConstantRector::class, [
+        'Doctrine\*'
+    ]);
```

Comment:
here `$classesToSkip` value need to be array:

```php
[
    'Doctrine\*'
]
```

with `$isBound` check, this still empty.